lib/charms/prometheus_k8s/v1/prometheus_remote_write.py

```diff
@@ -499,13 +501,21 @@ def _push_alerts_to_relation_databag(self, relation: Relation) -> None:
         if not self._charm.unit.is_leader():
             return
 
+        peer_relations = self._charm.model.get_relation(self._peer_relation_name)
+        peer_relation_units = peer_relations.units if peer_relations else []
```

Comment:
Would it be cleaner if we prepare the unit names here?


```suggestion
        unit_names = ({unit.name for unit in peer_relations.units} if peer_relations else set()) + {self._charm.unit.name}
```